src/client/camera.cpp

```diff
@@ -705,29 +690,27 @@ void Camera::drawNametags()
 		trans.multiplyWith1x4Matrix(transformed_pos);
 		if (transformed_pos[3] > 0) {
 			std::wstring nametag_colorless =
-				unescape_translate(utf8_to_wide(nametag->nametag_text));
+					unescape_translate(utf8_to_wide(nametag->nametag_text));
 			core::dimension2d<u32> textsize =
-				g_fontengine->getFont()->getDimension(
-				nametag_colorless.c_str());
+					g_fontengine->getFont()->getDimension(nametag_colorless.c_str());
 			f32 zDiv = transformed_pos[3] == 0.0f ? 1.0f :
-				core::reciprocal(transformed_pos[3]);
+													core::reciprocal(transformed_pos[3]);
```

Comment:
bad continuation

Reply:
Woah. That's a bit of a huge indentation. How'd that happen?

Reply:
It's aligning with the ternary statement, despite that being disabled